opengapps-packages.mk

```diff
@@ -179,13 +180,12 @@ ifeq ($(GAPPS_FORCE_PIXEL_LAUNCHER),true)
 GAPPS_EXCLUDED_PACKAGES += \
     GoogleHome
 
-ifneq ($(filter 25,$(call get-allowed-api-levels)),)
-ifneq ($(filter-out 26,$(call get-allowed-api-levels)),)
-DEVICE_PACKAGE_OVERLAYS += \
-    $(GAPPS_DEVICE_FILES_PATH)/overlay/pixelicons/25
-else
+ifneq ($(filter 26,$(call get-allowed-api-levels)),)
 DEVICE_PACKAGE_OVERLAYS += \
     $(GAPPS_DEVICE_FILES_PATH)/overlay/pixelicons/26
+else ifneq ($(filter 25,$(call get-allowed-api-levels)),)
```

Comment:
this part is not clear for me xD

Reply:
if the user wants the pixel launcher installed, then do the following:

if we are at api level 26 or greater, use the overlay `.../pixelicons/26`;
otherwise if we are at api level 25, use the overlay `.../pixelicons/25`

if we are at api level 25 or greater, install `PixelLauncherIcons`.